flapjack/commands.py

```diff
@@ -105,15 +106,16 @@ def execute(self, args):
 def find_remote_for_runtime(runtime, branch):
     """Search for the runtime in all configured remotes. Expensive check."""
     remotes_list = ext.flatpak('remotes', output=True).split('\n')[1:-1]
-    remotes = [line.split(' ', 1)[0] for line in remotes_list if line]
+    remotes = [re.split(' |\t', line, 1) for line in remotes_list if line]
     for candidate_remote in remotes:
-        runtimes_list = ext.flatpak('remote-ls', candidate_remote, '--runtime',
+        candidate_remote_name = candidate_remote[0]
+        runtimes_list = ext.flatpak('remote-ls', candidate_remote_name, '--runtime',
                                     '-d', output=True)
         runtimes_list = runtimes_list.split('\n')[1:]
         for line in runtimes_list:
             if not line:
                 continue
-            quad = line.split(' ', 1)[0]
+            quad = re.split(' |\t', line, 1)[0]
```

Comment:
`str.split()` without arguments does the split at anything considered whitespace, just what you want:
```
>>> "hello world \thow  are\n you?".split()
['hello', 'world', 'how', 'are', 'you?']
```

Reply:
So you can do: `line.split(maxsplit=1)[0]`

Reply:
Fixed